### ISSUE TITLE
How to install soketi in windows 11

### ISSUE THREAD

**Issue by amir1704** [soketi/pws#1117]
i have been install python but when i run` npm install -g @soketi/soketi`
error this: 
`npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
`

**Comment by iamjothees** [comment]
Same here.

Have you got any solution?

**Comment by lazosweb** [comment]
We had a similar issue. I think this is related to Node 20. Try to install Node 18 instead.

**Comment by iamjothees** [comment]
> We had a similar issue. I think this is related to Node 20. Try to install Node 18 instead.

Thanks for the quick response @lazosweb , 

Switching to  Node 18.19.1 worked! 👍

**Comment by amir1704** [comment]
yes, i run soketi in wsl(ubuntu) and find out it is because node version. but i need node in v20. when this bug fixed???

**Comment by iamjothees** [comment]
Hope that it will get fixed soon. 🤞

**Comment by iamjothees** [comment]
> yes, i run soketi in wsl(ubuntu) and find out it is because node version. but i need node in v20. when this bug fixed???

If so, I think you could opt for their docker image.

**Comment by lazosweb** [comment]
If you search in the issues tab for "Node 20" there is a post that someone mentioned in order to work you have to build uWebsockets.js manually. #1025

**Comment by chrispage1** [comment]
What about running it in Docker? This is a sure fire way of getting things working without too many dependencies - other than Docker of course..

**Comment by IkaroFox** [comment]
With me work in Docker , its simple and fast run a container

**Comment by whitespacecode** [comment]
> What about running it in Docker? This is a sure fire way of getting things working without too many dependencies - other than Docker of course..

There is no windows image of soketi for docker? [quay.io/soketi/soketi](https://quay.io/repository/soketi/soketi?tab=tags&tag=latest)
You would need to enable and use wsl on windows. Unless @rennokki could provide a solution for windows without enabling wsl?
